### PR TITLE
Align the calculator image to the center and add a back to top button

### PR DESCRIPTION
**Pull request by Devemmy01**
I aligned the calculator image to the center

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
   <!-- LineIcons -->
 
+  <link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
```
Comment (YoshithaRathnayake) [inline index.html:41]: Can you add this link tag in a new comment @Devemmy01 
````html
<!-- Boxicons -->

<link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
````

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (YoshithaRathnayake) [inline index.html:41]: Actually I want this scroll top button in dark/index.html and light/index.html files @Devemmy01 
No need in the index.html file

Comment (YoshithaRathnayake) [inline index.html:41]: Can you add this link tag in a new comment like below @Devemmy01 
````html
<!-- Boxicons -->

<link href='https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css' rel='stylesheet'>
````

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -53,10 +54,55 @@
 
   <title>CALCULATOR | YOSHITHA</title>
 
+  <style>
```
Comment (YoshithaRathnayake) [inline index.html:57]: Please can you move these css to the styles.css file @Devemmy01

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -53,10 +54,55 @@
...
+    visibility: visible;
+    bottom: 1.5rem;
+}
+
+  </style>
+
 </head>
 
 <body class="bg-img">
+
```
Comment (YoshithaRathnayake) [inline index.html:98]: Remove these unwanted lines @Devemmy01 
```suggestion
```

File: index.html [hunk 4]
```diff
@@ -133,8 +179,21 @@
...
+    scrollTop.classList.add("scroll-top");
+    else scrollTop.classList.remove("scroll-top")
+  }
+
+  window.addEventListener("scroll", scrollTop)
+  </script>
+
   <script src="https://apps.elfsight.com/p/platform.js" defer></script>
 
+
```
Comment (YoshithaRathnayake) [inline index.html:196]: These to
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 5]
```diff
@@ -53,10 +54,55 @@
...
+}
+
+  </style>
+
 </head>
 
 <body class="bg-img">
+
+
+  <!-- Scroll back to top icon -->
```
Comment (YoshithaRathnayake) [inline index.html:100]: Can you add these scroll to top button code after the navbar @Devemmy01

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Devemmy01) [reply index.html]: I'll make all changes now

File: index.html (hunk 5 above)
Comment (Devemmy01) [reply index.html]: All changes has been made @YoshithaRathnayake

File: assets/css/styles.css [hunk 6]
```diff
@@ -1,3 +1,38 @@
+    html{
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:1]: No need of this because its already there @Devemmy01

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 6 above)
Comment (Devemmy01) [reply assets/css/styles.css]: My bad. it's fixed

**Pull request closed without merging**